### PR TITLE
Scholarships namespace

### LINKED ISSUE #1081: add scholarship page namespace

### LINKED ISSUE #1082: list scholarships page namespace

### PR DESCRIPTION
**Pull request by edwingl10**
## Motivation and Context

Fixes #1081. Fixes #1082 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] User visible change (users will notice UI or functional changes)

## How Has This Been Tested?

- [ ] Existing or new tests cover my changes.
- [ ] Manually tested locally or on the Render PR Server.

## Previewing Changes

1. Click the `onrender.com` URL the **render `bot`** posted below.
2. Go to 'add scholarship' page (top section should be translated)
3. go scholarships page (tab title, end of results & no results found should be translated)
4. change language while on these pages

### CODE REVIEW

File: public/locales/es/addScholarship.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "titleTag": "Agrege una beca",
```
Comment (josueetcom) [inline public/locales/es/addScholarship.json:2]: Website actions in Spanish generally use the infinitive:
```suggestion
  "titleTag": "Agregar una beca",
```

File: public/locales/es/listScholarships.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "titleTag": "Buscar Becas",
+  "endOfResults": "Final de Resultados",
```
Comment (josueetcom) [inline public/locales/es/listScholarships.json:3]:
```suggestion
  "endOfResults": "Fin de los resultados",
```

File: public/locales/es/addScholarship.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+  "submitAScholarship": "Submitir una beca",
```
Comment (josueetcom) [inline public/locales/es/addScholarship.json:3]:
```suggestion
  "submitAScholarship": "Someter una beca",
```

**Review by josueetcom — APPROVED** [review]

File: public/locales/en/common.json [hunk 4]
```diff
@@ -48,7 +35,6 @@
     "subscribeForUpdates": "Subscribe for updates",
     "reachOut": "Reach out"
   },
-  "welcome": "Welcome",
```
Comment (josueetcom) [inline public/locales/en/common.json:51]: keep

File: public/locales/es/common.json [hunk 5]
```diff
@@ -48,7 +35,6 @@
     "subscribeForUpdates": "Suscríbete para actualizaciones",
     "reachOut": "Contacta con nosotros"
   },
-  "welcome": "Bienvenido",
```
Comment (josueetcom) [inline public/locales/es/common.json:51]: keep

**Review by josueetcom — APPROVED** [review]

File: public/locales/en/common.json (hunk 4 above)
Comment (edwingl10) [reply public/locales/en/common.json:51]: duplicate

File: public/locales/es/common.json (hunk 5 above)
Comment (edwingl10) [reply public/locales/es/common.json:51]: duplicate

**Pull request merged**